### PR TITLE
chore: Add a gnos token path

### PR DESCRIPTION
**Pull request by jinoosss**

### CODE REVIEW

File: packages/web/.env.example [hunk 1]
```diff
@@ -6,9 +6,11 @@ NEXT_PUBLIC_PACKAGE_POSITION_PATH="gno.land/r/demo/position"
...
-NEXT_PUBLIC_WRAPPED_GNOT_PATH="gno.land/r/demo/wugnot"
 NEXT_PUBLIC_PACKAGE_ROUTER_ADDRESS="g1pjtpgjpsn4hjfv2n4mpz8cczdn32jkpsqwxuav"
 NEXT_PUBLIC_PACKAGE_POOL_ADDRESS="g15z32w7txv6lw259xzhzzmwtwmcjjc0m6dqzh6f"
 NEXT_PUBLIC_PACKAGE_POSITION_ADDRESS="g10wwa53xgu4397kvzz7akxar9370zjdpwux5th9"
 NEXT_PUBLIC_PACKAGE_STAKER_ADDRESS="g1puv9dz470prjshjm9qyg25dyfvrgph2kvjph68"
-NEXT_PUBLIC_PACKAGE_GOVERNANCE_ADDRESS="g1kmat25auuqf0h5qvd4q7s707r8let5sky4tr76"
\ No newline at end of file
+NEXT_PUBLIC_PACKAGE_GOVERNANCE_ADDRESS="g1kmat25auuqf0h5qvd4q7s707r8let5sky4tr76"
+NEXT_PUBLIC_WRAPPED_GNOT_PATH="gno.land/r/demo/wugnot"
+NEXT_PUBLIC_GNOS_TOKEN_PATH="gno.land/r/demo/gnos"
```
Comment (dongwon8247) [inline packages/web/.env.example:15]: GNOS -> GNS

File: packages/web/src/repositories/pool/pool-repository-impl.ts [hunk 2]
```diff
@@ -36,6 +36,8 @@ const WRAPPED_GNOT_PATH = process.env.NEXT_PUBLIC_WRAPPED_GNOT_PATH || "";
 const POOL_PATH = process.env.NEXT_PUBLIC_PACKAGE_POOL_PATH || "";
 const POSITION_PATH = process.env.NEXT_PUBLIC_PACKAGE_POSITION_PATH || "";
 const POOL_ADDRESS = process.env.NEXT_PUBLIC_PACKAGE_POOL_ADDRESS || "";
+const GNOS_TOKEN_PATH = process.env.NEXT_PUBLIC_GNOS_TOKEN_PATH || "";
```
Comment (dongwon8247) [inline packages/web/src/repositories/pool/pool-repository-impl.ts:39]: GNOS -> GNS

File: packages/web/src/repositories/pool/pool-repository-impl.ts [hunk 3]
```diff
@@ -267,7 +269,7 @@ export class PoolRepositoryImpl implements PoolRepository {
   }
 
   private static makeApproveGnosTokenMessage(caller: string) {
-    return this.makeApproveTokenMessage("gno.land/r/gns", "500", caller);
+    return this.makeApproveTokenMessage(GNOS_TOKEN_PATH, CREATE_POOL_FEE, caller);
```
Comment (dongwon8247) [inline packages/web/src/repositories/pool/pool-repository-impl.ts:272]: GNOS -> GNS

**Review by dongwon8247 — APPROVED** [review]

File: packages/web/.env.example (hunk 1 above)
Comment (jinoosss) [reply packages/web/.env.example]: Thanks for letting us know what we missed.

**Comment by r3v4s** [comment]
BTW, its `gns`, not `gnos`

**Pull request merged**